models/dex/dex_trades.sql

```diff
@@ -41,6 +41,7 @@ list of models using old generic test, due to multiple versions in one model:
 ,'nomiswap_bnb_trades'
 ,'gmx_trades'
 ,'biswap_bnb_trades' 
+,'dodo_bnb_trades'
```

Comment:
Hi @LeviScoffie , since dodo already exists on ethereum, do you think you can combine dodo_bnb and dodo_ethereum trades before adding to dex.trades?

Reply:
Yes, Let me do that ASAP @Hosuke